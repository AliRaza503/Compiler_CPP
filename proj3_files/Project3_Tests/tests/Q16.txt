{
    switch(a){
        case 1:{
            b = * c a;
        }
        case 2:{
             x = z;
        }
        case 3: {
            z = <= z z;
        }

    }
}