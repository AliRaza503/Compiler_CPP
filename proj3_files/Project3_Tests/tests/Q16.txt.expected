a: int #
b: int #
c: int #
x: bool #
z: bool #
